res/problems/dependency-heaven/problem.md

```diff
@@ -0,0 +1,57 @@
+Write an HTTP **server** that serves JSON data when it receives a GET request to `/reverse`, `/swapcase` and `/titleize`. 
+
+The GET data will contain a single parameter `data` which you will need to manipulate depending on the endpoint.
+
+## /reverse  
+
+A request with `data = "PHP School is awesome!"` should return the response:
```

Comment:
Is this string meant to look like the POST body? I don't think there is spaces either side of assignment